src/website/views/components/modal.pug

```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
         </div>
       </div>
     </div>
+
+h4 Scrollable
+p.-text
+  | If the content height is bigger than the content modal container
+  | height, we can make the content scrollable.
+h5 Requirements
+p.-text
+  | Content Modal must have the<code>max-height</code> defined.
+
+.example.-mb--3
+  .a-backdrop.-p--6.-position--relative.-z--0(style="border-top-left-radius:3px;border-top-right-radius:3px;")
+    .a-modal(style="margin-top:0;")
+      .a-modal__header
+        h3.a-modal__title Modal Title
+        button(class='a-btn -icon -close')
+          .a-btn__content
+            i.a-icon.icon-x
+      .a-modal__content(style='max-height:400px;')
```

Comment:
Max-height was increased but we need more content to trigger the overflow-y. Can you add an extra lorem ipsum paragraph? 5 or 6 paragraphs should do it with a -mt--0 on the first-child.

Reply:
ok